ios/ThreadManager.m

```diff
@@ -47,34 +47,30 @@ -(void) checkAndSendEvent:(NSString*)name body:(id)body
   }
 
   int threadId = nextThreadId++;
-
-  NSString *jsFileSlug = [name containsString:@"./"]
-    ? [name stringByReplacingOccurrencesOfString:@"./" withString:@""]
-    : name;
-
-  // Bundled JavaScript is placed within the resources directory.
-  NSURL *bundledThreadURL = [[NSBundle mainBundle] URLForResource:[NSString stringWithFormat:@"%@", [jsFileSlug lastPathComponent]] withExtension:@"jsbundle"];
-
+    
+  NSURL *threadURL;
+
+#ifdef DEBUG
+  threadURL = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:name];
```

Comment:
We currently generate debug builds that allow us to perform instrumentation tests on BrowserStack, where we can't run metro.

I will try to upload a release build and see if the unit tests succeed, then this assumption should be okay. If not, we might need to make a check like `if debug && ping metroPort`?

Reply:
https://github.com/ExodusMovement/template-wallet/pull/144/files#diff-4e85e9488a33c7fd412722451e70a345346e4602119e94eda8dc782a50c8ed79R21-R51

Reply:
Works! 😀

Reply:
Do I need to do something?